modules/web-api.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python
+
+import requests
+import argparse
+import datetime
+from datetime import timedelta
+
+API_REPORTS = '/api/v2/reports'
+API_RESULTS = '/api/v2/results'
+API_STATUS = '/api/v2/status'
+
+strerr = ''
+num_excp_expand = 0
+server_expire = None
+NAGIOS_RESULT = 0
+
+
+def errmsg_from_excp(e):
+    global strerr, num_excp_expand
+    if isinstance(e, Exception) and getattr(e, 'args', False):
+        num_excp_expand += 1
+        if not errmsg_from_excp(e.args):
+            return strerr
+    elif isinstance(e, dict):
+        for s in e.iteritems():
+            errmsg_from_excp(s)
+    elif isinstance(e, list):
+        for s in e:
+            errmsg_from_excp(s)
+    elif isinstance(e, tuple):
+        for s in e:
+            errmsg_from_excp(s)
+    elif isinstance(e, str):
+        if num_excp_expand <= 5:
+            strerr += e + ' '
+    elif isinstance(e, int):
+        if num_excp_expand <= 5:
+            strerr += str(e) + ' '
+
+
+def checkUnusedReports(tenant, hostname, reportName):
+
+    """ We have a number of reports that are not used anymore.
+        In order not to check these reports we define which one
+        to ignore
+        Args:
+            tenant: the tenant to test
+            hostname: we use the hostname to check for devel, or production
+            reportName: the name of the report to check
+    """
+    #reports we dont use in production
+    EGI_prod = ['Cloud', 'Critical-Fedcloud', 'Fedcloud']
+    EGI_devel = ['Cloud', 'Critical-Fedcloud', 'Fedcloud']
+
+    if (tenant == 'EGI' and hostname == 'api.argo.grnet.gr'):
```

Comment:
i added a new jira ticket so as to update the probe this week.. 
https://jira.argo.grnet.gr/browse/ARGO-822